contrib/platform/src/com/sun/jna/platform/win32/WinNT.java

```diff
@@ -3061,4 +3169,76 @@ public SYSTEM_LOGICAL_PROCESSOR_INFORMATION(Pointer memory) {
             return FIELDS;
         }
     }
+
+    /**
+     * Contains information about a range of pages in the virtual address space of a process.
+     */
+    class MEMORY_BASIC_INFORMATION64 extends Structure {
+        public static final List<String> FIELDS = createFieldsOrder("baseAddress", "allocationBase",
+                "allocationProtect", "__alignment1", "regionSize", "state", "protect", "type", "__alignment2");
+
+        /**
+         * A pointer to the base address of the region of pages.
+         */
+        public ULONGLONG baseAddress;
+
+        /**
+         * A pointer to the base address of a range of pages allocated by the VirtualAlloc function.
+         * The page pointed to by the BaseAddress member is contained within this allocation range.
+         */
+        public ULONGLONG allocationBase;
+
+        /**
+         * The memory protection option when the region was initially allocated. This member can be
+         * one of the memory protection constants or 0 if the caller does not have access.
+         */
+        public DWORD allocationProtect;
+
+        public DWORD __alignment1;
+
+        /**
+         * The size of the region beginning at the base address in which all pages have identical
+         * attributes, in bytes.
+         */
+        public ULONGLONG regionSize;
+
+        /**
+         * The state of the pages in the region.
+         */
+        public DWORD state;
+
+        /**
+         * The access protection of the pages in the region. This member is one of the values listed
+         * for the AllocationProtect member.
+         */
+        public DWORD protect;
+
+        /**
+         * The type of pages in the region.
+         */
+        public DWORD type;
+        public DWORD __alignment2;
+
+        public static class ByReference extends MEMORY_BASIC_INFORMATION64 implements Structure.ByReference {
+            public ByReference() {
+            }
+
+            public ByReference(Pointer memory) {
+                super(memory);
+            }
+        }
+
+        public MEMORY_BASIC_INFORMATION64() {
+        }
```

Comment:
Don't leave this empty - add call to (implicit) _super()_ as this enables setting a breakpoint in the constructor.